text/2020-09-03-factored-prefix.md

```diff
@@ -0,0 +1,71 @@
+# Factored Key Prefix
+
+## Summary
+
+TiKV provides a facility where the client can set a key prefix that will be used for all keys in its requests.
+
+## Motivation
+
+TiKV users need to use key prefixes to avoid conflicts.
+In keeping with DRY we should not require a program to add the same prefix everywhere.
+This will also reduce the payload sent to TiKV.
```

Comment:
If the prefix is small enough compare to the total payload, we may not see clear win with the approach.

Reply:
It is not a goal so we can remove it.